src/Domain/Purchase/SubscriptionService.php

```diff
@@ -97,6 +100,31 @@ public function legacyCancel(string $packageName, string $token, CancellationTyp
         $this->client->sendRequest($request);
     }
 
+    public function legacyDefer(string $packageName, string $token, SubscriptionDeferralInfo $deferralInfo): Time
```

Comment:
The method signature does not match the API design.

Endpoint variables:
- packageName
- subscriptionId
- token

In addition to the body:
- deferralInfo

The return data type could be `DeferSubscriptionResponse`